### PR TITLE
More SD observables added to the SD response task.

### PR DESCRIPTION
**Pull request by klapidus**

### CODE REVIEW

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDropResponse.h [hunk 1]
```diff
@@ -68,6 +72,17 @@ class AliAnalysisTaskSoftDropResponse : public AliJetResponseMaker {
...
+ 
+  void               Decluster(const fastjet::PseudoJet& jet);
+  fastjet::ClusterSequence* Recluster(const AliEmcalJet* jet);
+  
+  void               FillZgRgVectors(const AliEmcalJet* jet);
+
+  Int_t nsdsteps;
+  std::vector<Float_t> zg_values; //n_SD ordering
+  std::vector<Float_t> rg_values; //n_SD ordering
+  std::vector<Int_t>   SDsteps_values;
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDropResponse.h]: You should:
- update your ClassDef
- rename your variables to start with "f" to keep with convention
- Set the streaming properties (ie `//!<!` or `///<`)

**Comment by raymondEhlers** [comment]
+test

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDropResponse.h (hunk 1 above)
Comment (raymondEhlers) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDropResponse.h]: According to the discussion in #5173, you don't need to update your ClassDef if these are just transient. But I think the other comments are still important

Comment (klapidus) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDropResponse.h]: Pardon my ignorance, where the new streamer instructions are explained? I find only old style //!

Comment (raymondEhlers) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDropResponse.h]: I probably should have included a link. See [here](https://dberzano.github.io/alice/doxygen/#data_members)

**Comment by raymondEhlers** [comment]
+1 looks good!

**Pull request merged**